### PR TITLE
Update v2 workflows to current impl

### PR DESCRIPTION
**Pull request by chee**
--

### CODE REVIEW

File: templates/workflows/test-origami-component.yml [hunk 1]
```diff
@@ -5,18 +5,14 @@ jobs:
...
-        GITHUB_TOKEN: ${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}
-        ORIGAMI_PULL_REQUEST: ${{ github.event.pull_request.html_url }}
+    - run: npm install -g npm@^7
+    - run: npm install -g origami-build-tools@prerelease
+    - run: obt install
+      continue-on-error: true
+    - run: obt verify
+      continue-on-error: true
+    - run: obt test
+      continue-on-error: true
```
Comment (JakeChampion) [inline templates/workflows/test-origami-component.yml:18]: This is really nice!

**Review by JakeChampion — APPROVED** [review]

File: templates/workflows/test-origami-component.yml (hunk 1 above)
Comment (chee) [reply templates/workflows/test-origami-component.yml:18]: i think @notlee did this!

Comment (notlee) [reply templates/workflows/test-origami-component.yml:18]: chee did this! :)

Comment (notlee) [reply templates/workflows/test-origami-component.yml:18]: no?

Comment (chee) [reply templates/workflows/test-origami-component.yml:18]: must have been @JakeChampion

Comment (JakeChampion) [reply templates/workflows/test-origami-component.yml:18]: 😂

**Pull request merged**